### ISSUE TITLE
Crash due to mismatched labels and Deep Learning Effect output

### ISSUE THREAD

**Issue by oreillyp** [hugofloresgarcia/audacity#94]
### Bug description

If a `multichannel` Deep Learning Effect produces more output channels than the number of `labels` specified in the metadata, Audacity crashes without an error message.

### Steps to reproduce

Create a simple pass-through model, with `multichannel` set to `True` and a single output label:
```
class PassThrough(WaveformToWaveformBase):
    
    def do_forward_pass(self, x: torch.Tensor) -> torch.Tensor:
        return self.model(x)

effect = PassThrough(torch.nn.Identity())

metadata = {
    ...
    'labels': ['output'],
    'effect_type': 'waveform-to-waveform',
    'multichannel': True,
}
```

Script / serialize the model, deploy via HuggingFace, and load in Audacity as a Deep Learning Effect on a stereo input track; this should produce two channels of audio output, but only one output label is specified in the metadata.

### Expected behavior

Ideally, Audacity would apply the effect, create the required number of output tracks, and automatically assign labels if too few were provided in the metadata. Barring that, an informative error message would be better than a straight crash.

### Audacity Version

current master / alpha / nightly build

### Operating system

macOS

**Comment by aldo-aguilar** [comment]
error does not reproduce on Ubuntu 22.04.1

**Comment by hugofloresgarcia** [comment]
@aldo-aguilar what's the behavior on Ubuntu?

**Comment by aldo-aguilar** [comment]
`rebase-3.2.3` has solved this issue
